datafusion/src/physical_plan/functions.rs

```diff
@@ -1023,6 +1098,18 @@ fn signature(fun: &BuiltinScalarFunction) -> Signature {
             Signature::Exact(vec![DataType::LargeUtf8, DataType::Int64]),
         ]),
         BuiltinScalarFunction::ToTimestamp => Signature::Uniform(1, vec![DataType::Utf8]),
+        BuiltinScalarFunction::ToTimestampMillis => Signature::Uniform(
+            1,
+            vec![DataType::Utf8, DataType::LargeUtf8, DataType::Int64],
```

Comment:
These signatures claim to support arguments of type`LargeUtf8` but `LargeUtf8` isn't handled in the implementations above. I think we should make them consistent (I don't have a preference about actually supporting `LargeUtf`, btw)

Reply:
@alamb I actually don't understand what LargeUTF is about, but since `ToTimestamp` only supports regular Utf8, is it OK if all of the timestamp functions just support regular Utf8?

Reply:
> @alamb I actually don't understand what LargeUTF is about,

`Utf8`uses i32 to store offsets and thus can store strings up to 2GB.  `LargeUtf8`  stores string offsets using u64s and thus can support strings that are larger than 2GB. I do not know how widely `LargeUtf8` is used in practice. 

> is it OK if all of the timestamp functions just support regular Utf8? 

yes I think so